### PR TITLE
use portable path api

### PR DESCRIPTION
**Pull request by i4ki**
yeah, I'm a windows user hahaha

### CODE REVIEW

File: vendor.go [hunk 1]
```diff
@@ -109,24 +110,22 @@ func getGoPath() string {
 	if gopath != "" {
 		return gopath
 	}
-	home := os.Getenv("HOME")
-	if home == "" {
-		fmt.Println("no GOPATH env var found and no HOME to infer GOPATH from")
-		os.Exit(1)
-	}
-	return filepath.Join(home, "go")
+	u, err := user.Current()
```
Comment (katcipis) [inline vendor.go]: Much nicer =)

File: vendor.go [hunk 2]
```diff
@@ -109,24 +110,22 @@ func getGoPath() string {
...
-	srcpkgpath := filepath.Join(gohome, "src", pkg)
+func createDir(dir string) {
+	err := os.MkdirAll(dir, 0774)
+	abortonerr(err, fmt.Sprintf("creating dir[%s]", dir))
+}
 
 func vendorPackages(depsGoHome string, projectdir string) {
-	depsrootdir := path.Join(depsGoHome, "src")
-	projectVendorPath := path.Join(projectdir, "vendor")
+	depsrootdir := filepath.Join(depsGoHome, "src")
```
Comment (katcipis) [inline vendor.go:124]: Can't find a difference between path and filepath, path always uses "/" and filepath is portable ?

**Review by katcipis — APPROVED** [review]
LGTM

File: vendor.go (hunk 2 above)
Comment (i4ki) [reply vendor.go:124]: yep

**Pull request merged**